Makefile

```
@@ -434,6 +434,14 @@ android-devices: export TARGET := android-sdk
 android-devices: ##@other Invoke adb devices
 	adb devices
 
+android-pull-geth: export VERSION ?= debug
```

By using a more exact smaller shell startup will be faster.
```suggestion
android-pull-geth: export TARGET := android-sdk
android-pull-geth: export VERSION ?= debug
```